istio-cni/values.yaml

```diff
@@ -1,5 +1,6 @@
 cni:
   hub: gcr.io/istio-release
+  image: install-cni
```

Comment:
And maybe remove this - the default values ( | default ...)  will be used. 

I don't think this is a common use case - the official release uses the default names, and advanced users who build  their  own can still override the setting. 

IMO using Kustomize or something similar would be a cleaner approach.

Reply:
Yeah I would like to use kustomize as well but our system right now is strictly helm 😅.

Will update to move default into the chart though.

Reply:
done